site/collections/_projects/sustainability-leadership-kosova.md

```diff
@@ -5,8 +5,33 @@ subtitle: Used data to influence an ongoing governmental draft on deforestation
 image: "/images/SKL-project.png"
 ---
 
-<a href="https://www.slkosova.org/">Sustainability Leadership Kosova</a> (SLK) empowers individuals, businesses and organisations in leadership with the aim to cultivate innovative solutions to sustainability challenges. With the help of a local GIS expert, Faruk Foniqi (UN Habitat), SLK has uncovered the rate of deforestation (including illegal deforestation) in Kosovo
+# The Challenge
+[Sustainability Leadership Kosova (SLK)](https://www.slkosova.org/) is a non-profit organization dedicated to empowering individuals, businesses, and organizations to lead and integrate sustainability solutions in Kosovo. Founded in 2019 by Indira Kartallozi and a group of passionate collaborators, SLK aims to drive system change across the economy, people, and nature to create a sustainable, healthy, fair, and prosperous society.
 
-Faruk has used satellite imagery using a GIS model for his analysis and has even made a predictive analysis for future deforestation. The aim of this project was to use this data to influence an ongoing governmental draft on deforestation regulation strategy as well as empower Civil Society Organisations, NGOs, and activists to increase pressure on the government to act.
+One of SLK’s significant challenges has been addressing deforestation in Kosovo. With the assistance of local GIS expert Faruk Foniqi, SLK identified the rate of deforestation, including illegal activities, using satellite imagery and GIS analysis. This data aimed to influence governmental draft regulations on deforestation and empower Civil Society Organisations, NGOs, and activists to pressure the government for action.
 
```

Comment:
In SCL, I created separate headers for _The Partner Organisation_ and _The Challenge_. I think that would fit here very well as well.

Reply:
In addition, I would change "One of SLK's significant challenges" to "One of SLK's focuses has been addressing deforestation in Kosovo.". The word "challenge" should be reserved for the challenge that CorrelAid addresses. 

Or "An important topic for SLK has been deforestation in Kosovo" / "An important problem SLK has been addressing is" etc.